mesheryctl/internal/cli/root/system/update.go

```diff
@@ -40,13 +37,12 @@ var updateCmd = &cobra.Command{
 		return utils.PreReqCheck(cmd.Use, "")
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-
-		if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
-			if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
-				return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
-			}
+		userResponse := utils.AskForConfirmation("Meshery container images are going to be updated. Are you sure you want to continue")
```

Comment:
This is desired behavior only if users are also notified that they are running an old version and asked to update. Otherwise, we have intentionally set the behavior here to autoupdate to ensure that users are always using the latest and greatest.

A subscription to a release channel is .... just that - a _subscription_. If they are on `stable-latest`, then they should receive _latest_. Unless the user has pinned their version in their meshconfig, autoupdates should be happening under their subscription.

Reply:
i.e for implementing this, 

if the user is on the stable-latest channel, if they run `mesheryctl system update`, it should just go and update without asking the user for confirmation.

If the user is on a specific version, then we should ask for confirmation.

Is that the case?

Reply:
Perfect, yes.

Reply:
We may get feedback in the future that asking the user to update when they have intentionally pinned to a specific version is not ideal. Let's wait for people to give that feedback, though.

Reply:
Yep. @aneeshnema could you please check this out?

Reply:
fixed ✔